packages/realm_dart/lib/src/cli/install/install_command.dart

```diff
@@ -100,7 +99,7 @@ class InstallCommand extends Command<void> {
     await versionFile.writeAsString(version.toString());
   }
 
-  Future<Package> getPackage(String name) async {
```

Comment:
Why do you prefer to traffic in typeless `String`s over `Package`?

Reply:
Because we don't use anything from the package except for the root folder. I think structuring the code that way makes it more explicit - it doesn't force the reader to figure out what the Package API is and/or wonder what exactly it is being used for.

Reply:
Could we use `Directory`/`File`?

Reply:
We could - just don't see much value in it, since `path.join` takes strings and the Directory/File API doesn't seem to do any validations either, so it'll be just wrapping the paths. I guess one benefit is that it's going to be a bit more explicit about what the return type is, but I feel this is reasonably well achieved by the method naming.

Reply:
I guess I'm just a bit allergic to use _typeless_ values, hence why I made it a `Package` in the first place. What are you actually trying to fix with this PR. It is not immediately apparent to me.